perf/benchmark/runner/runner.py

```diff
@@ -314,7 +362,8 @@ def validate(job_config):
             return False
         exp_type = required_fields[k]
         if not isinstance(job_config[k], exp_type):
-            print("expecting type of parameter {} to be {}, got {}".format(k, exp_type, type(job_config[k])))
+            print("expecting type of parameter {} to be {}, got {}".format(
+                k, exp_type, type(job_config[k])))
             return False
     return True
```

Comment:
@oschaaf I've deleted this func, since I could not found where we are using this.

Reply:
This part of the diff looks like a formatting change to the original code that I didn't mean to touch; it existed at some point in the commit list of this PR when my editor reformatted it, but it is not around in the latest commit of this PR where I cleaned up the diff.

Having said that, I agree we won't need the formatting change.

ps - Looking more closely, the function (`validate(job_config)`) is used on master, but was [renamed](https://github.com/istio/tools/blob/master/perf/benchmark/runner/runner.py#L379) to `validate_job_config`.

Reply:
no worries, we are not using it. 

Btw, I was trying to rebase this PR with master, but, it's really annoying, since we already have 31 commits there. By resolving those conflict files directly on Github works easy. HAHA

Reply:
wait, you mean validate_job_config, let me check

Reply:
yeah, just a renaming

Reply:
actually, I was referring to the rc(command) func, which we are not using, but I don't know why it's commented out on this validate() func.